0.download_data/download_jump_data.ipynb

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
If you add the file you generated `jump_dataset.csv` with, I recommend also changing the name of this file to include a sequential prefix.

E.g.
0.generate_jump_dataset_manifest.ipynb
1.download_jump_data.ipynb

(also, shouldn't `1.download_jump_data.ipynb` actually be named `1.download_jump_single_cell_profiles.ipynb`? Someone else might think that you're downloading the images and you're not.